ovos_padatious/train_data.py

```
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ovos_padatious.util import tokenize, expand_parentheses, remove_comments
+from ovos_padatious.util import tokenize, expand_lines, expand_parentheses, remove_comments
```

Ruff (0.8.0)

`ovos_padatious.util.expand_parentheses` imported but unused

Remove unused import

(F401)